src/windowsdesktop/Directory.Build.props

```diff
@@ -1,8 +1,19 @@
 <Project>
-
   <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
 
   <PropertyGroup>
+    <!-- WiX v5 Configuration -->
+    <UseWix5>true</UseWix5>
+    
+    <!-- WiX v5 PDB Type Fix -->
+    <WixToolsetCompilerPdbType>full</WixToolsetCompilerPdbType>
+    <WixToolsetLinkerPdbType>full</WixToolsetLinkerPdbType>
+    
+    <!-- Ensure Wixpack generation is enabled for all build types -->
+    <WixCreateWixPackOutput Condition="'$(OutputType)' == 'Bundle'">true</WixCreateWixPackOutput>
+    <RunWixpackTargets Condition="'$(ContinuousIntegrationBuild)' == 'true'">true</RunWixpackTargets>
```

Comment:
Why would this not be created in non-CI builds? If you wanted to conditionalize it, I would do so on signing.